gcn_email_service/__init__.py

```diff
@@ -214,3 +205,5 @@ def main():
     connect_as_consumer()
     subscribe_to_topics()
     recieve_alerts()
+
+main()
```

Comment:
Remove:

```suggestion
```

There is already a console entry point that calls the main function.